lib/core-common/src/utils/__tests__/paths.test.ts

```diff
@@ -3,54 +3,27 @@ import { normalizeStoryPath } from '../paths';
 
 describe('paths - normalizeStoryPath()', () => {
   it('returns a path starting with "./" unchanged', () => {
-    const filename = `./src/Comp.story.js`;
+    const filename = `.${path.sep}${path.join('src', 'Comp.story.js')}`;
     expect(normalizeStoryPath(filename)).toEqual(filename);
   });
 
   it('returns a path starting with "../" unchanged', () => {
-    const filename = `../src/Comp.story.js`;
+    const filename = path.join('..', 'src', 'Comp.story.js');
     expect(normalizeStoryPath(filename)).toEqual(filename);
   });
 
   it('adds "./" to a normalized relative path', () => {
-    const filename = `src/Comp.story.js`;
-    expect(normalizeStoryPath(filename)).toEqual(`./${filename}`);
+    const filename = path.join('src', 'Comp.story.js');
+    expect(normalizeStoryPath(filename)).toEqual(`.${path.sep}${filename}`);
   });
 
   it('adds "./" to a hidden folder', () => {
-    const filename = `.storybook/Comp.story.js`;
-    expect(normalizeStoryPath(filename)).toEqual(`./${filename}`);
+    const filename = path.join('.storybook', 'Comp.story.js');
+    expect(normalizeStoryPath(filename)).toEqual(`.${path.sep}${filename}`);
   });
 
   it('adds "./" to a hidden file', () => {
     const filename = `.Comp.story.js`;
-    expect(normalizeStoryPath(filename)).toEqual(`./${filename}`);
-  });
-
-  describe('windows paths', () => {
-    it('returns a path starting with ".\\" unchanged', () => {
-      const filename = `.\\src\\Comp.story.js`;
-      expect(normalizeStoryPath(filename)).toEqual(filename);
-    });
-
-    it('returns a path starting with "..\\" unchanged', () => {
-      const filename = `..\\src\\Comp.story.js`;
-      expect(normalizeStoryPath(filename)).toEqual(filename);
-    });
-
-    it('adds ".{path.sep}" to a normalized relative path', () => {
-      const filename = `src\\Comp.story.js`;
-      expect(normalizeStoryPath(filename)).toEqual(`.${path.sep}${filename}`);
-    });
-
-    it('adds ".{path.sep}" to a hidden folder', () => {
-      const filename = `.storybook\\Comp.story.js`;
-      expect(normalizeStoryPath(filename)).toEqual(`.${path.sep}${filename}`);
-    });
-
-    it('adds ".{path.sep}" to a hidden file', () => {
-      const filename = `.Comp.story.js`;
-      expect(normalizeStoryPath(filename)).toEqual(`.${path.sep}${filename}`);
-    });
```

Comment:
I removed these because they were written in order to test windows-style paths on non-windows github machines.  Now that we can use `path` in the tests to actually create windows paths on windows, these duplicates are no longer needed.